src/MSAAdvanced.sol

```diff
@@ -369,7 +369,8 @@ contract MSAAdvanced is
 
         // checks if already initialized and reverts before setting the state to initialized
         _initModuleManager();
-
+        _addStorageBase(MODULEMANAGER_STORAGE_LOCATION);
```

Comment:
is there a way to only write this on 7702? the easiest case would be to only write if msg.sender == address(this) but I wonder if there's another way

Reply:
is that what 7702 returns these days?

Reply:
or `if(extcodehash(address(this)) == keccak256(0xef01)`

Reply:
yeah lets go with either one of these but we need to look out for 7702 changing yet again lol

Reply:
hahah. yeah, this is being discussed already :)

Reply:
if it changes we can use keccak256(extcodecopy[0:1])